sdk/default_variable_parser.py

```
@@ -15,6 +19,27 @@ def parse(self,
               variant_name: str,
               config: str) -> Optional[dict]:
         try:
-            return jsons.loads(config, dict)
-        except DeserializationError:
-            return None
+            result = json.loads(config)
+            if isinstance(result, dict):
+                return result
+            return result
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)